src/app/forms/controls/text.html

```diff
@@ -0,0 +1,58 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h2>Text Input</h2>
+
+<form class="clr-form">
+    <div class="clr-form-control">
+        <label for="basic" class="clr-control-label">
+            Basic
+        </label>
+        <div class="clr-control-container">
+            <div class="clr-input-wrapper">
+                <input type="text" id="basic" placeholder="Enter value here" class="clr-input">
```

Comment:
Did we try using the `input` without the wrapper to see if it renders correctly? Also it should render the same if the user doesn't add `type="text"` because the default type is `text`. This was a bug on our side before.

Reply:
This doesn't work completely, but making a backlog note to address in a future iteration.